api/README.md

```diff
@@ -1,44 +1,75 @@
-# REST HTTP API INSTRUCTIONS
+# Verification API
```

Comment:
Could we fix the formatting for the other example requests/responses in this file? For example, the indentation in the requests/responses for the "Example (URI or shared volume path)" section look a bit off.

Reply:
Done.